woocommerce/woocommerce
# add: blueprint settings dropzone

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #53679.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Turn on the blueprint feature flag using WC Test Helper
2. Go to WooCommerce > Settings > Advanced > Blueprint
3. Observe that there is a Blueprint uploader dropzone
4. Scroll down and export a blueprint
5. After the blueprint is downloaded, open dev console and go to network tab
6. Drag blueprint.json into the dropzone
7. Observe that there is a post request to `wc-admin/blueprint/queue`

https://github.com/user-attachments/assets/1dd52949-2e98-473d-99c9-b912addf196c






### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
